x/evm/keeper/bank_extension.go

```diff
@@ -206,6 +206,7 @@ func (bk NibiruBankKeeper) ForceGasInvariant(
 		return err
 	}
 
+	ctx = ctx.WithGasMeter(sdk.NewInfiniteGasMeter())
```

Comment:
Should we reset ctx's gas meter to the original `gasMeterBefore` after `AfterOp`?

Reply:
Realized that ctx is passed by value, so any updates to it in this scope won't affect the ctx from the outer scope.